gremlin-go/driver/resources/error-messages/en.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Considering the naming here, we should renaming the other one to logger-message/en.json to match.